pkgs/top-level/all-packages.nix

```
@@ -3596,7 +3596,9 @@ with pkgs;
 
   reg = callPackage ../tools/virtualization/reg { };
 
-  river = callPackage ../applications/window-managers/river { };
+  river = callPackage ../applications/window-managers/river {
+    zig = zig_0_8_1;
+  };
```

```suggestion
  river = callPackage ../applications/window-managers/river { };
```

river 0.1.2 calls for zig 0.9.0 now, otherwise LGTM

---

Isn't river 0.1.2 still in staging? And this PR is in `master`.

---

I decided not to target staging, and wait for the wlroots update to hit master.

---

Then it should be `zig = zig_0_8_1;`, so this PR could be merged before the wlroots update hits the master.